scripts/generators/tsconfig.js

```diff
@@ -34,6 +38,35 @@ const tsPkgs = [
   ...getTsPkgs("codemods"),
 ];
 
+function sourceDeps(packageDir) {
+  const files = globby.sync(`src/**/*.ts`, {
+    cwd: packageDir,
+    onlyFiles: true,
+    dot: true,
+    ignore: ["**/node_modules/**"],
+  });
+  const result = new Set();
+  for (const file of files) {
+    const filename = path.join(packageDir, file);
+    const source = fs.readFileSync(filename, { encoding: "utf8" });
+
+    const ast = babel.parseSync(source, {
+      ast: true,
+      filename,
+    });
+
+    traverse(ast, {
+      ImportDeclaration(path) {
+        const importSource = path.node.source.value;
+        if (importSource.startsWith("@babel/")) {
+          result.add(importSource);
+        }
+      },
+    });
+  }
```

Comment:
I wonder if we could speed this up with

```js
for (const [importSource] of source.matchAll(/(?<=from\s*")@babel\/[^"/]+/g)) {
  result.add(importSource);
}
```

this doesn't work for generic JS, but it should be ok for our own source files.

Reply:
that actually not only faster - it also covers additional cases when when having  `export {} from ""` statement

updated the PR